src/main/java/com/csc530/sat/SMTVariable.java

```diff
@@ -0,0 +1,43 @@
+package com.csc530.sat;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.ToString;
+
+@Builder
+@AllArgsConstructor
+@ToString
+public class SMTVariable<C> {
```

Comment:
this could just be `Variable` now imho 
I should have selected that name in the first place :woman_shrugging: